app/views/CancellationReasonView.scala.html

```diff
@@ -36,13 +37,14 @@
 
         @errorSummary(form.errors)
 
+        @headingCaption(lrn.value)
+
         @inputCharacterCount(
             field = form("value"),
             label = messages("cancellationReason.title"),
-            maxLength = maxLength,
-            caption = Some(lrn.value)
+            maxLength = maxLength
         )
 
         @button(messageKey = "site.submit", attributes = Map("id" -> "submit"))
     }
-}
+}@import views.html.components.HeadingCaption
```

Comment:
Not sure if it matter too much with scope, but think this should be at the top with the other imports